### DISCUSSION TITLE
Restrict branch updates to schedule

### DISCUSSION THREAD

**Discussion by btxbtx** [renovatebot/renovate#18497] [Request Help] [answered]
I have Renovate running as a CronJob every hour on kubernetes.  A few of my `packageRules` indicate a schedule of once weekly.  As expected, the branches are only created once a week.  However, after creation, the branches are updated hourly (every time the CronJob runs) -- rerunning the ci, even when there are no changes.

How can I tell Renovate to only attempt to update a branch following a schedule?

**Comment by viceice** [comment, marked as answer]
there's a config option update not scheduled, check the docs.

**Comment by btxbtx** [comment]
Okay, I found it thanks.  Maybe it is a good idea to link to that in the `schedule` docs, which currently make `schedule` seems as if it works as I expected it to originally:

```
The schedule option allows you to define times of week or month for Renovate updates. Running Renovate around the clock can be too "noisy" for some projects. To reduce the noise you can use the schedule config option to limit the time frame in which Renovate will perform actions on your repository.
```

**Reply by rarkins** [reply]
PRs welcome
